### ISSUE TITLE
关于文章的问题---期待回复

### ISSUE THREAD

**Issue by AAAceerr** [machengcheng2016/CPE-LTSSL#5]
文中描述：
To address this issue, inspired by (Cui et al. 2022), we
propose Classwise Batch Normalization (CBN) mechanism
to realize classwise feature processing. CBN involves three
different BN layers to re-scale the features output by the
shared encoder g(·), denoted by {BNHMT, BNMT, BNT}.
Concretely, BNHMT processes the features belonging to all
classes, while BNMT processes features in medium and tail
classes (denoted by M and T ), and BNT only processes features in tail classes. Formally, the cross entropy loss of expert Ei on unlabeled data xu in Eq. (4) is re-formulated as
（第四节experiments的上面）
这里写道：具体地，BNHMT处理属于所有类的特征，而BNMT处理中间类和尾类（由M和T表示）中的特征，而BNT仅处理尾类中的特征。
这个是什么意思呢？代码里面，三个分类头对logits做的是一样的操作呀，都是三等分然后求平均？是代码出错了吗？
